### ISSUE TITLE
Question about bundles

### ISSUE THREAD

**Issue by xJohnx117** [YaLTeR/MouseTweaks#115]
I've wanted to know if holding right click to store lots of individual items with a bundle is exclusive to 1.19.2 or higher, because I feel like the tweak is really handy!

**Comment by xJohnx117** [comment]
Well I would say that bundles didn't come with that feature, do you think mods that directly add vanilla bundles do that, like quark?

**Comment by YaLTeR** [comment]
Yeah, they were added to the game but aren't obtainable. Mouse Tweaks hardcodes this behavior to the vanilla bundle item since there isn't really a generic way to tell if some item acts like a vanilla bundle.

**Comment by xJohnx117** [comment]
So some mods might be conflicting with Mousetweaks?
